package.json

```diff
@@ -43,7 +43,7 @@
     "typescript": "4.0.2"
   },
   "dependencies": {
-    "@handlebars/parser": "^2.0.0",
+    "@handlebars/parser": "~2.0.0",
```

Comment:
Why do we need to pin this? This seems not good...

Reply:
For some reason, the change was breaking. I'm not sure what the issue is exactly and have not had time to dig in

Reply:
sure but since nothing else changed package.json / yarn.lock in this PR you probably can just delete the changes and `yarn` without issue

Reply:
So I did that, tests failed, this change did result in `yarn.lock` changes and frozen lockfile threw

Reply:
Running `yarn` on master does not require any `yarn.lock` changes, and `@handlebars/parser` is at 2.0.0. 

I've pushed 60f2dec72018e5e5fae7bd1701b5d9bd98434170 fixing this for you.